### ISSUE TITLE
px deploy cluster type check fails for k3s

### ISSUE THREAD

**Issue by bpschmitt** [pixie-io/pixie#460]
**Describe the bug**
I'm attempting to deploy Pixie into a `k3s` cluster which is listed as supported in the [Pixie docs](https://docs.px.dev/installing-pixie/requirements/#kubernetes-local-development-environments).

The `Cluster type is supported` check passes, but then fails later saying the cluster type isn't in the list of known supported types.

I'm able to manually proceed with the deploy but it would be helpful to clean up this logic so that it proceeds automatically.

```
# px deploy
Pixie CLI

Running Cluster Checks:
 ✔    Kernel version > 4.14.0 
 ✔    Cluster type is supported 
 ✔    K8s version > 1.16.0 
 ✔    Kubectl > 1.10.0 is present 
 ✔    User can create namespace 
 ✕    Cluster type is in list of known supported types  ERR: Cluster type is not in list of known supported cluster types. Please see: https://docs.px.dev/installing-pixie/requirements/
 Some cluster checks failed. Pixie may not work properly on your cluster. Continue with deploy? (y/n) [y] : y
Installing Vizier version: 0.11.3
Generating YAMLs for Pixie
Deploying Pixie to the following cluster: default
Is the cluster correct? (y/n) [y] : y
Found 1 nodes
 ✔    Installing OLM CRDs 
 ✔    Deploying OLM 
 ✔    Deploying Pixie OLM Namespace 
 ✔    Installing Vizier CRD 
 ✔    Deploying OLM Catalog 
 ✔    Deploying OLM Subscription 
 ✔    Creating namespace 
 ✔    Deploying Vizier 
 ✔    Waiting for Cloud Connector to come online 
Waiting for Pixie to pass healthcheck
 ✔    Wait for PEMs/Kelvin 
 ⠴    Wait for healthcheck 
 ```

**Expected behavior**
Successfull install of Pixie into a `k3s` cluster without manual intervention.

**Screenshots**
If applicable, add screenshots to help explain your problem. Please make sure the screenshot does not contain any sensitive information such as API keys or access tokens.

**Logs**
Please attach the logs by running the following command:
```
./px collect-logs
```

**App information (please complete the following information):**

```
Linux kubernetes-vm 5.13.0-1031-gcp #37~20.04.1-Ubuntu SMP Thu Jun 9 13:25:43 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux


Client Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.6+k3s1", GitCommit:"418c3fa858b69b12b9cefbcff0526f666a6236b9", GitTreeState:"clean", BuildDate:"2022-04-28T22:16:18Z", GoVersion:"go1.17.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.6+k3s1", GitCommit:"418c3fa858b69b12b9cefbcff0526f666a6236b9", GitTreeState:"clean", BuildDate:"2022-04-28T22:16:18Z", GoVersion:"go1.17.5", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by htroisi** [comment]
Looks like we are doing a strict match for `k3s` after the `+` split, instead of a regex looking for `k3s`: https://github.com/pixie-io/pixie/blob/c808313f1374e7b62255e667526fe3855d2f37e7/src/pixie_cli/pkg/utils/checks.go#L146

**Comment by htroisi** [comment]
Fixed in https://github.com/pixie-io/pixie/commit/2f10816e08924ae71c37adf2bbb193cae4925fc7

**Comment by vihangm** [comment]
This should be fixed in cli release [v0.7.14](https://github.com/pixie-io/pixie/releases/tag/release%2Fcli%2Fv0.7.14)
